android/src/ti/map/TiUIMapView.java

```diff
@@ -1092,4 +1145,19 @@ protected void onViewCreated() {
 		// keep around for backward compatibility
 	}
 
+	@Override
+	public boolean onClusterClick(Cluster<TiClusterMarker> cluster) {
+		LatLngBounds.Builder builder = LatLngBounds.builder();
+		for (TiClusterMarker item : cluster.getItems()) {
+			builder.include(item.getPosition());
+		}
+		final LatLngBounds bounds = builder.build();
+
+		if (map != null) {
+			try {
+				map.animateCamera(CameraUpdateFactory.newLatLngBounds(bounds, 100));
+			} catch (Exception e) {}
```

Comment:
I think adding `e.printStackTrace()` could be useful for debugging potential issues around this code.

Reply:
done